app/scripts/modules/openstack/serverGroup/configure/wizard/templateSelection.html

```diff
@@ -0,0 +1,65 @@
+<div modal-page ng-controller="openstackServerGroupTemplateSelectionCtrl as templateSelection">
+  <modal-close></modal-close>
+  <div ng-if="state.loaded">
+    <div class="modal-header">
+      <h3>Template Selection</h3>
+    </div>
+    <div class="modal-body">
+      <form class="form-horizontal">
+        <div class="form-group">
+          <div class="col-md-4 col-md-offset-1 sm-label-left">
+            <b>Copy configuration from</b>
+          </div>
+        </div>
+        <div class="form-group">
+          <div class="col-md-6 col-md-offset-1">
+            <ui-select ng-model="command.viewState.template"
+                       class="form-control input-sm">
+              <ui-select-match placeholder="Select...">
+              <span ng-if="!$select.selected.label">
+                <account-tag account="$select.selected.account"></account-tag>
+                <span ng-if="$select.selected.serverGroup">{{$select.selected.serverGroupName}}</span>
+                ({{$select.selected.region}})
+              </span>
+                <span ng-if="$select.selected.label">{{$select.selected.label}}</span>
+              </ui-select-match>
+              <ui-select-choices repeat="template in templates | anyFieldFilter: {label: $select.search, serverGroupName: $select.search}">
+                <h5 ng-if="!template.label"><account-tag account="template.account"></account-tag> {{template.cluster}} ({{template.region}})</h5>
+                <h5 ng-if="template.label">{{template.label}}</h5>
+                <div ng-if="template.serverGroup">
+                  <b>Most recent server group: </b> {{template.serverGroupName}}
+                </div>
+              </ui-select-choices>
+            </ui-select>
+          </div>
+        </div>
+        <div class="form-group" ng-if="command.viewState.customTemplateMessage">
+          <p class="col-md-10 col-md-offset-1" ng-bind-html="command.viewState.customTemplateMessage"></p>
+        </div>
+        <div class="form-group" ng-if="command.viewState.template.serverGroup" style="margin-top: 20px">
+          <div class="col-md-10 col-md-offset-1 well">
+            These fields <strong>will be</strong> copied over from the most recent server group:
+            <ul>
+              <li>account, namespace, cluster name (stack, details)</li>
+              <li>load balancers</li>
+              <li>security groups</li>
```

Comment:
Security groups are not being surfaced in the wizard here - is that something you folks want to tackle later or as part of this PR?